cypress/integration/newsBodySpec.js

```diff
@@ -43,8 +43,11 @@ describe('Article Body Tests', () => {
     placeholderImageLoaded(getElement('figure div').eq(0));
   });
 
-  it('should have a visible image without a caption', () => {
-    visibleImageNoCaption(getElement('figure').eq(0));
+  it('should have a visible image without a caption, and also not be lazyloaded', () => {
+    const firstFigure = getElement('figure').eq(0);
+
+    visibleImageNoCaption(firstFigure);
+    firstFigure.within(() => getElement('noscript').should('not.exist'));
```

Comment:
Rather than checking for the `noscript` image which is part of the lazyload logic but not explicitly ties to lazyloaing this should check there is not lazyload div EG: `<div class="lazyload">`

Reply:
Once an image has lazyloaded, there is no evidence that it was lazyloaded other than the `noscript`. There isn't a wrapping div that indicates it's been lazyloaded afaict

Reply:
Hmmm yeah that is a bit annoying. Could you try added a class name to the Image in the LazyLoad ternary.

This allows us to seperate the logic of LazyLoading and noscript, ideally we don't want to assume test logic on a related element, albeit as closely related as noscipt and lazyload are in this situation.

Reply:
Update: At a quick glance because we set  `<Image>` to a const EG:`const imageToRender = <Image alt={alt} src={src} width={width} />;` I'm not sure we can inject a class name 

EG:
`<Image className="willLazyload">`

So might be that there isn't a nice way to handle this without an overly complex refactor